sys-process/oomd/oomd-0.5.0-r1.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="oomd is an userspace Out-Of-Memory (OOM) killer for Linux systems."
```

We don't repeat `${PN}` in description, and don't put `.` at the end of this descriptive text either.
```suggestion
DESCRIPTION="An userspace Out-Of-Memory (OOM) killer for Linux systems"
```
or
```suggestion
DESCRIPTION="Userspace Out-Of-Memory (OOM) killer for Linux systems"
```